src/cpr_sdk/pipeline_general_models.py

```diff
@@ -109,3 +110,13 @@ class ExecutionData(BaseModel):
     """Data unique to a step functions execution that is required at later stages."""
 
     input_dir_path: str
+
+
+class DocUpdateConfig(BaseModel):
+    """
+    Config for updates not defined as part of IdentifyUpdates.
+
+    reprocess_updates: list of document ids to reprocess.
+    """
+
+    reprocess_updates: list[str]
```

Comment:
Whats the thinking behind this being an sdk model rather than a pipeline model? Are we likely to use this anywhere else?

Reply:
So we will want to pass the updates in to the pipeline trigger through the ui and pass them through into the identify updates batch job, so those two places would be using this object. 

(Obviously this may not be necessary when / if you implement identify updates in the pipeline repo)

Reply:
Ah, fair!

Reply:
If you do implement we can look to clear this down!